src/ad-hoc-visualizations/issues/visualization.tsx

```diff
@@ -56,10 +56,26 @@ export const IssuesAdHocVisualization: VisualizationConfiguration = {
         }),
     getIdentifier: () => AdHocTestkeys.Issues,
     visualizationInstanceProcessor: () => VisualizationInstanceProcessor.nullProcessor,
-    getNotificationMessage: selectorMap =>
-        isEmpty(selectorMap)
-            ? 'Congratulations!\n\nAutomated checks found no issues on this page.'
-            : null,
+    getNotificationMessage: (selectorMap, key, warnings) => {
+        const iframeCoda =
+            '\nThere are iframes in the target page. Use FastPass or Assessment to provide additional permissions.';
+
+        if (isEmpty(selectorMap)) {
+            if (isEmpty(warnings)) {
+                return 'Congratulations!\n\nAutomated checks found no issues on this page.';
+            }
+
+            return `No automated checks issues found.${iframeCoda}`;
+        }
+
+        const noIssuesFound = 'Automated checks found issues.';
```

Comment:
This variable name should probably be "issues found", no?

Reply:
shameful display :( I was moving code around, changing the "direction" of the conditionals... I guess this was named on a previous iteration. Thanks for catching this.